src/Ancestor/Ancestor.DataAccess/3.DAO/MSSqlDao.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Ancestor.DataAccess.DAO
 {
-    public class MSSqlDao : DataAccessObject, IDataAccessObject
+    public class MSSqlDao : BaseAbstractDao, IDataAccessObject
```

Comment:
Because BaseAbstractDao was already implement IDataAccessObject
So IDataAccessObject can be removed from here.